tools/protoxform/protoprint.py

```diff
@@ -204,7 +204,15 @@ def CamelCase(s):
   file_block = '\n'.join(['syntax = "proto3";\n', package_line])
 
   options = descriptor_pb2.FileOptions()
+
   options.java_outer_classname = CamelCase(os.path.basename(file_proto.name))
+  for msg in file_proto.message_type:
```

Comment:
If this loop is concerning we could alternatively just special case the three impacted connection pool proto files.

Reply:
Fine by me as is.